### ISSUE TITLE
Tasic Implementation - heatmap plot data

### ISSUE THREAD

**Issue by mah533** [dkobak/finer-tsne#1]
Hello,

In Tasic implementation, the file "sample_heatmap_plot_data.csv" is needed which must be built using **Sample Heatmaps** feature on *https://portal.brain-map.org/atlases-and-data/rnaseq*. 

Apparently this feature has been deprecated or at least I couldn't find it. Could you please advise.

Sincerely,

**Comment by dkobak** [comment]
Right. Thanks. You can get it here: https://raw.githubusercontent.com/berenslab/mini-atlas/master/data/raw/allen/tasic2018/sample_heatmap_plot_data.csv
